docs/content/latest/quick-start/build-apps/java/ysql-jdbc-ssl.md

```diff
@@ -0,0 +1,226 @@
+headerTitle: Build a Java application
+linkTitle: Java
+description: Build a sample Java application with the PostgreSQL JDBC Driver and use the YSQL API to connect to and interact with YugabyteDB via SSL/TLS.
+menu:
+  latest:
+    parent: build-apps
+    name: Java
+    identifier: java-2
+    weight: 550
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="/latest/quick-start/build-apps/java/ysql-jdbc" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - JDBC
+    </a>
+  </li>
+  <li >
+    <a href="/latest/quick-start/build-apps/java/ysql-jdbc-ssl" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - JDBC SSL/TLS
+    </a>
+  </li>
+  <li >
+    <a href="/latest/quick-start/build-apps/java/ysql-spring-data" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - Spring Data JPA
+    </a>
+  </li>
+  <li>
+    <a href="/latest/quick-start/build-apps/java/ycql" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+  <li>
+    <a href="/latest/quick-start/build-apps/java/ycql-4.6" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL (4.6)
+    </a>
+  </li>
+</ul>
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. If you are new to YugabyteDB, you can download, install, and have YugabyteDB up and running within five minutes by following the steps in [Quick start](../../../../quick-start/).
+- Java Development Kit (JDK) 1.8, or later, is installed. JDK installers for Linux and macOS can be downloaded from [OpenJDK](http://jdk.java.net/), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/).
+- [Apache Maven](https://maven.apache.org/index.html) 3.3 or later, is installed.
+- [OpenSSL](https://www.openssl.org/) is installed.
+
+## Set up SSL certificates for Java applications
+
+To build a Java application that connects to YugabyteDB over an SSL connection, you need the root certificate (`ca.crt`), and node certificate (`yugabytedb.crt`) and key (`yugabytedb.key`) files. If you have not generated these files, follow the instructions in [Create server certificates](../../../../secure/tls-encryption/server-certificates/).
+
+1. Download the certificate (`yugabytedb.crt`, `yugabytedb.key`, and `ca.crt`) files (see [Copy configuration files to the nodes](../../../../secure/tls-encryption/server-certificates/#copy-configuration-files-to-the-nodes)).
+
+1. If you do not have access to the system `cacerts` Java truststore you can create your own truststore.
+
+    ```sh
+    $ keytool -keystore ybtruststore -alias ybtruststore -import -file ca.crt
+    ```
+
+1. Verify the `yugabytedb.crt` client certificate with `ybtruststore` by running the following command:
```

Comment:
You could probably eliminate all the "by running the following command" instances